### PR TITLE
[OBMIN-176] Updated CSS to Styled-components (file AddGoods) 

### PR DESCRIPTION
**Pull request by smilaya**

### CODE REVIEW

File: src/main/resources/react/src/components/common/checkbox/styles.js [hunk 1]
```diff
@@ -31,7 +31,7 @@ const Label = styled.label`
   margin-left: ${(props) => (props.distanceBetween ? '9px' : '22px')};
   font-size: ${({ fontSize }) => fontSize || '14px'};
   line-height: 17px;
-  vertical-align: middle;
+  /* vertical-align: middle; */
```
Comment (Wolshebnik) [inline src/main/resources/react/src/components/common/checkbox/styles.js:34]: delete pls

File: src/main/resources/react/src/pages/AddGoods/AddGoods.jsx [hunk 2]
```diff
@@ -96,7 +96,7 @@ const AddGoods = () => {
     description: Yup.string()
       .max(255, getTranslatedText('errors.max255'))
       .default(() => description),
-    locationId: Yup.string()
+    locationId: Yup.number()
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/AddGoods/AddGoods.jsx:99]: change pls to string()

File: src/main/resources/react/src/pages/AddGoods/exchange/index.jsx [hunk 3]
```diff
@@ -61,47 +62,48 @@ const Exchange = ({ exchangeList, setExchange }) => {
...
+      <Styles.Explanation>
         ({getTranslatedText('addAdv.enterPhrase')})
-      </p>
+      </Styles.Explanation>
 
-      <div className={`change_wrapper ${getBorderClassName(border, error)}`}>
+      <Styles.ChangeWrapp
+        styles={getBorderClassName}
+        borderValue={border}
+        errorValue={error}
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/AddGoods/exchange/index.jsx:83]: sort by length

File: src/main/resources/react/src/pages/AddGoods/photo-files/add-file-input/index.jsx [hunk 4]
```diff
@@ -4,7 +4,7 @@ import { useField } from 'formik';
 import dropsPng from 'assets/img/drag-n-drop.png';
 
 import { ErrorDisplay } from '../../error-display';
-import { FilesLabel, Input, SpanAdd, Image, WrapError } from './styles';
+import * as Styles from './styles';
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/AddGoods/photo-files/add-file-input/index.jsx:7]: sort by length

File: src/main/resources/react/src/pages/AddGoods/selection-section/index.jsx [hunk 5]
```diff
@@ -10,15 +10,7 @@ import { FormikCheckBox } from 'components/common/formik';
...
-
-import {
-  TitleH3,
-  Sections,
-  AddChoose,
-  InputText,
-  SectionsItem,
-  ItemDescription,
-} from './styles';
+import * as Styles from './styles';
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/AddGoods/selection-section/index.jsx:13]: sort by length

File: src/main/resources/react/src/pages/AddGoods/selection-section/styles.js [hunk 6]
```diff
@@ -48,7 +48,12 @@ const InputText = styled.input`
   }
 `;
 
+const Star = styled.span`
+  color: red;
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/AddGoods/selection-section/styles.js:52]: maybe should theme
${({theme}) => theme.colors......}
should be red color

File: src/main/resources/react/src/pages/AddGoods/show-select-item/index.jsx [hunk 7]
```diff
@@ -8,16 +8,7 @@ import { getTranslatedText } from 'components/local/localization';
...
-  Image,
-  DropItems,
-  SelectItem,
-  WrapSelect,
-  PlaceHolder,
-  SelectLabel,
-  SelectTitle,
-  AnimatedLabel,
-} from './styles';
+import * as Styles from './styles';
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/AddGoods/show-select-item/index.jsx:11]: sort by length

File: src/main/resources/react/src/pages/AddGoods/styles.js [hunk 8]
```diff
@@ -0,0 +1,159 @@
+import styled from 'styled-components';
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/AddGoods/styles.js:1]: past space pls

File: src/main/resources/react/src/pages/AddGoods/styles.js [hunk 9]
```diff
@@ -0,0 +1,159 @@
...
+  justify-content: space-between;
+  margin-bottom: 50px;
+  flex-wrap: wrap;
+  & :last-child {
+    color: #383838;
+    width: 350px;
+  }
+`;
+
+export const WrapItem = styled.div``;
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/AddGoods/styles.js:44]: if don't use delete pls

File: src/main/resources/react/src/pages/AddGoods/styles.js [hunk 10]
```diff
@@ -0,0 +1,159 @@
...
+  padding: 10px;
+  width: 100%;
+  min-height: 150px;
+  font-size: 16px;
+  line-height: 24px;
+  outline: none;
+  resize: none;
+  border: 1px solid #bcbcbc;
+  border-radius: 2px;
+  caret-color: ${({ theme: { colors } }) => colors.activeColor};
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/AddGoods/styles.js:75]: ${({theme}) => theme.colors.activeColor}

File: src/main/resources/react/src/pages/AddGoods/styles.js [hunk 11]
```diff
@@ -0,0 +1,159 @@
...
+    content: '';
+    width: 1px;
+    height: 16px;
+    background-color: black;
+  }
+  &::before {
+    top: 50%;
+    left: 50%;
+    transform: translate(-50%, -50%) rotate(45deg);
+  }
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/AddGoods/styles.js:137]: space

File: src/main/resources/react/src/pages/AddGoods/styles.js [hunk 12]
```diff
@@ -0,0 +1,159 @@
...
+  transition: all 0.4s ease-in-out;
+
+  &:before,
+  &:after {
+    position: absolute;
+    content: '';
+    width: 1px;
+    height: 16px;
+    background-color: black;
+  }
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/AddGoods/styles.js:132]: space

File: src/main/resources/react/src/pages/AddGoods/styles.js [hunk 13]
```diff
@@ -0,0 +1,159 @@
...
+    transform: translate(-50%, -50%) rotate(-45deg);
+  }
+`;
+
+export const BackButtons = styled.div`
+  display: flex;
+  padding: 5px;
+  text-transform: uppercase;
+  cursor: pointer;
+  transition: all 0.2s ease-in-out;
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/AddGoods/styles.js:150]: space

File: src/main/resources/react/src/pages/AddGoods/styles.js [hunk 14]
```diff
@@ -0,0 +1,159 @@
...
+  &:hover {
+    ${Cross} {
+      transform: rotate(180deg);
+    }
+  }
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/AddGoods/styles.js:155]: space

File: src/main/resources/react/src/pages/AddGoods/styles.js [hunk 15]
```diff
@@ -0,0 +1,159 @@
...
+  font-weight: 600;
+  margin-bottom: 10px;
+  line-height: 40px;
+`;
+
+export const WrapItems = styled.div`
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 50px;
+  flex-wrap: wrap;
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/AddGoods/styles.js:37]: space

File: src/main/resources/react/src/pages/AddGoods/exchange/styles.js [hunk 16]
```diff
@@ -0,0 +1,102 @@
...
+  height: 32px;
+  margin: 7px;
+  font-family: 'Roboto', sans-serif;
+  line-height: 24px;
+  border: none;
+  outline: none;
+  caret-color: #00c0ff;
+  &:focus {
+    border-color: hsl(0, 0%, 44%);
+  }
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/AddGoods/exchange/styles.js:96]: space

File: src/main/resources/react/src/pages/AddGoods/exchange/styles.js [hunk 17]
```diff
@@ -0,0 +1,102 @@
...
+  position: relative;
+  min-width: 262px;
+  box-sizing: border-box;
+  height: 32px;
+  margin: 7px;
+  font-family: 'Roboto', sans-serif;
+  line-height: 24px;
+  border: none;
+  outline: none;
+  caret-color: #00c0ff;
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/AddGoods/exchange/styles.js:93]: space

File: src/main/resources/react/src/pages/AddGoods/exchange/styles.js [hunk 18]
```diff
@@ -0,0 +1,102 @@
...
+  }
+
+  &:before {
+    position: absolute;
+    content: '';
+    width: 1px;
+    height: 11px;
+    background: #fff;
+    transform: rotate(-45deg);
+  }
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/AddGoods/exchange/styles.js:76]: space

File: src/main/resources/react/src/pages/AddGoods/exchange/styles.js [hunk 19]
```diff
@@ -0,0 +1,102 @@
...
+  &:hover {
+    transform: rotate(90deg);
+  }
+`;
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/AddGoods/exchange/styles.js:80]: space

File: src/main/resources/react/src/pages/AddGoods/exchange/styles.js [hunk 20]
```diff
@@ -0,0 +1,102 @@
...
+  position: absolute;
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+  top: calc(50% - 8px);
+  width: 16px;
+  height: 16px;
+  right: 8px;
+  cursor: pointer;
+
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/AddGoods/exchange/styles.js:57]: delete space

File: src/main/resources/react/src/pages/AddGoods/photo-files/add-file-input/index.jsx [hunk 21]
```diff
@@ -29,27 +29,27 @@ const AddFileInput = ({ onChange }) => {
...
       onDragStart={(e) => dragStartHandler(e)}
       onDragLeave={(e) => dragLeaveHandler(e)}
     >
-      <Input
+      <Styles.Input
         multiple
         type="file"
         name="file"
         accept=".png, .jpg, .jpeg, .gif"
         onChange={onChange}
```
Comment (Wolshebnik) [inline src/main/resources/react/src/pages/AddGoods/photo-files/add-file-input/index.jsx:44]: I forgot, sort by length

**Review by Wolshebnik — APPROVED** [review]

**Review by cloudmenson — APPROVED** [review]

**Review by Wolshebnik — APPROVED** [review]

**Pull request merged**